app/waf/src/controllers/adc.controller.ts

```diff
@@ -37,15 +37,16 @@ export class AdcController {
       },
     },
   })
-  async create(@requestBody() adc: Partial<Adc>): Promise<Adc> {
+  async create(@requestBody() adc: Partial<Adc>): Promise<AdcResponse> {
     if (!adc.id) {
       adc.id = uuid();
     }
 
     try {
-      return await this.adcRepository.create(adc);
+      let data = await this.adcRepository.create(adc);
+      return new AdcResponse(data);
```

Comment:
All `Promise<xxxx>` functions should return `Promise.resolve(...)` or `Promise.reject(...)`.

Reply:
Done